index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Create Mobile Version</title>
+</head>
+<body>
+    <main>
+        
+        <header>
+            <h1 class="logo">Welcome</h1>
+     
+            <nav class="menu-bar">
+                <div class="bar"></div>
+                <div class="bar"></div>
+                <div class="bar"></div>
+                <div class="bar"></div>
```

Comment:
- [ ] You should use icons (such as fontawsome fa-icons) or svg images from the figma design (by exporting the `Icons/Icon - Menu` component under `Toolbar`) instead of using `<div>` to draw the hamburger button yourself.

- [ ] You should change the size of the hamburger bar because it is big as compared to the design

Reply:
changed implemented on the hamburger bar.